src/Hydrator/MoneyHydrator.php

```diff
@@ -20,7 +20,7 @@ class MoneyHydrator implements HydratorInterface
     public function extract($object)
     {
         return [
-            'amount' => $object->getAmount(),
+            'amount' => $object->getAmount() / 100,
```

Comment:
Quite a few actually. Here's a list: https://support.stripe.com/questions/which-zero-decimal-currencies-does-stripe-support
Beside this being a BC-break, I really want discourage this change. Working with the smallest unit available is very common practice when performing payments. It's also less error prone (rounding problems).